### PR TITLE
chore(release): update chart w/ breaking changes from 0.6.0

### PR DESCRIPTION
**Pull request by dogonthehorizon**
### Summary

Update this helm chart w/ the latest (including breaking changes) for the armory-agent.

### Notes

Creating a draft PR to capture early feedback, I am testing this now in a dev cluster.

### CODE REVIEW

File: templates/deployment.yml [hunk 1]
```diff
@@ -28,9 +26,9 @@ spec:
       - name: {{ .Values.imagePullSecrets }}
       {{- end }}
       containers:
-      - image: armory/kubesvc:0.6.0-rc.40
+      - image: armory/agent-k8s:{{ .appVersion }}
```
Comment (dogonthehorizon) [inline templates/deployment.yml:29]: I noticed that this wasn't using appVersion before and I was curious if that was intentional?

File: Chart.yaml [hunk 2]
```diff
@@ -10,4 +10,4 @@ version: 0.1.2
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.6.0-rc.40"
+appVersion: "0.6.0"
```
Comment (german-muzquiz) [inline Chart.yaml:13]: Armory Cloud support has not been released yet in a stable version. For using these helm charts with the rebranding the only supported version is `0.7.0-rc.1`

File: templates/deployment.yml (hunk 1 above)
Comment (german-muzquiz) [reply templates/deployment.yml:29]: I don't think it was intentional, it's good to use appVersion here

**Review by german-muzquiz — APPROVED** [review]

**Comment by dogonthehorizon** [comment]
Tested and validated that install w/ agent/hube mode works correctly 👍

**Pull request merged**